### PR TITLE
Added a setting for player name

### PR DESCRIPTION
**Pull request by daniep01**
Added a setting player_name which is displayed at the top of the schedule page and in html <title> to help identify which screen you are editing when multiple boxes are in use.

### CODE REVIEW

**Comment by vpetersson** [comment]
@daniep01 - looks sensible. However:

 * Let's change it to 'Player Name' instead of system name
 * The linting failed as you can see in the Travis run, so let's fix that too.

File: server.py [hunk 1]
```diff
@@ -303,7 +303,8 @@ def recover():
 @route('/')
 @auth_basic
 def viewIndex():
-    return template('index')
+    system_name=settings['system_name']
```
Comment (vpetersson) [inline server.py]: What happens if this setting is absent?

Comment (daniep01) [inline server.py]: It has a default in settings.py so should be always set

**Comment by daniep01** [comment]
The travis error line is
socket.error: [Errno 98] Address already in use
I'm not sure how to resolve this - doesn't seem to be related to this change.

**Comment by vpetersson** [comment]
@daniep01 the issue is not that (that one is suppressed). The issue is:

```
0.85s$ find . ! -path "*/rtmplite/*" -name \*.py -exec pep8 --ignore=E402,E501,E731 {} +
./server.py:306:16: E225 missing whitespace around operator
```

**Comment by daniep01** [comment]
Thanks, white space issue now resolved and travis happy.

**Comment by daniep01** [comment]
As requested in #349  and #485.

**Comment by vpetersson** [comment]
Just one last thing @daniep01 - what happens if the setting is absent in `setting.ini`? Since there is no migration, it will be absent on all nodes who are upgrading.

**Comment by daniep01** [comment]
I can't find `setting.ini` in the code, do you mean `screenly.conf`?

**Comment by vpetersson** [comment]
@daniep01 yes - sorry meant `screenly.conf`. In theory, it should use the default, but let's double check.

**Comment by vpetersson** [comment]
Cool. Thanks!

**Comment by vpetersson** [comment]
@antonmolodykh could you just take a quick look at this as well so that we have a second pair of eyes.

**Comment by antonmolodykh** [comment]
LGTM

**Pull request merged**